eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Templates.php

```diff
@@ -41,7 +41,7 @@ public function addSemanticConfig(NodeBuilder $nodeBuilder)
     public function preMap(array $config, ContextualizerInterface $contextualizer)
     {
         foreach ($config['siteaccess']['groups'] as $group => $saArray) {
-            if (isset($config['system'][$group][static::NODE_KEY])) {
+            if (isset($config['system'][$group][static::NODE_KEY]) && !empty($config[$contextualizer->getSiteAccessNodeName()][$group][static::NODE_KEY])) {
```

Comment:
Why don't you just replace `isset()` by `!empty()`? `empty()` implies an `isset()` test